api/controller.py

```diff
@@ -961,6 +961,7 @@ def feed(self, lane_identifier, feed_class=OPDSAcquisitionFeed):
         )
         return feed.as_response(
             max_age=int(max_age) if max_age else None,
+            requested_content_type=flask.request.headers.get("Accept"),
```

Comment:
I think we need to do something a bit more sophisticated then this to properly support [content negotiation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation). Some more detail here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

This is a valid accept header:
`Accept: application/atom+xml;profile=opds-catalog;kind=acquisition, application/opds+json;q=0.9, */*;q=0.8`

Which would indicate that the client wants OPDS 1.x response. But we would return an OPDS 2.x response.

It seems like flask provides some parsing of accept headers that we should be using in [Request.accept_mimetypes](https://flask.palletsprojects.com/en/2.3.x/api/#flask.Request.accept_mimetypes), rather then using the header directly.